crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__declarations_missing.snap

```diff
@@ -18,19 +17,27 @@ error[Compatibility E01001]: missing public declaration
 4 │ module upgrades::func_ {
   │                  ^^^^^ public function 'fun_to_be_removed' is missing
   │
-  = public function is missing expected public function 'fun_to_be_removed', but found none
   = public functions are part of a module's public interface and cannot be removed or changed during an upgrade.
   = add missing public function 'fun_to_be_removed' back to the module 'func_'.
 
+error[Compatibility E01001]: missing public declaration
+  ┌─ /fixtures/upgrade_errors/declaration_errors_v2/sources/func.move:7:9
+  │
+7 │     fun fun_to_lose_public(): u64 {
+  │         ^^^^^^^^^^^^^^^^^^ Function 'fun_to_lose_public' has lost its public visibility
+  │
+  = Functions are part of a module's public interface and cannot be changed during an upgrade.
+  = Restore the original function's 'public' visibility for function 'fun_to_lose_public'.
+
 error[Compatibility E01001]: missing public declaration
   ┌─ /fixtures/upgrade_errors/declaration_errors_v2/sources/struct.move:4:18
   │
 4 │ module upgrades::struct_ {
   │                  ^^^^^^^ struct 'StructToBeRemoved' is missing
   │
-  = struct is missing expected struct 'StructToBeRemoved', but found none
   = structs are part of a module's public interface and cannot be removed or changed during an upgrade.
   = add missing struct 'StructToBeRemoved' back to the module 'struct_'.
 
+The following modules are missing from the new package: 'missing_module'
```

Comment:
Likewise, this should probably be the second message after an overall "things went wrong" diagnostic, as it's a higher level/larger error than specific compatibility issues from a file, and it's easy to lose it among the detail.